### PR TITLE
Specify worksheet (by index) in cells merge

### PR DESCRIPTION
**Pull request by tntrex**
Currently, if you merge cells on multiple worksheets, all merges will be applied on all worksheets. So tables will look broken in Excel. This is hotfix PR for the problem.

P.S. Sorry about PR without test case. I have no time for it right now.

### CODE REVIEW

**Comment by Slamdunk** [comment]
Hi, thank you for your contribution, it looks promising.

Take your time to write tests, there's no hurry 😉

**Comment by tntrex** [comment]
Hello @Slamdunk 

I've just repushed branch: added test case, fixed codestyle and phpstan noticies

File: src/Writer/XLSX/Options.php [hunk 1]
```diff
@@ -43,7 +62,13 @@ public function mergeCells(
         int $bottomRightColumn,
         int $bottomRightRow
     ): void {
+        $writer = $this->getWriter();
```
Comment (Slamdunk) [inline src/Writer/XLSX/Options.php]: Stateful `Options` would be confusing to use and hard to maintain.

We can simply extend `Options::mergeCells` API to have a 5th parameter for the `$sheetIndex`, defaulted to `0` to keep backward compatibility.

**Review by Slamdunk — CHANGES_REQUESTED** [review]
Overall it looks good, just simplify the API and we can :ship:

File: src/Writer/XLSX/Options.php (hunk 1 above)
Comment (tntrex) [reply src/Writer/XLSX/Options.php]: Done refactoring

**Review by Slamdunk — APPROVED** [review]

**Comment by Slamdunk** [comment]
Thank you @hustlahusky

**Pull request merged**